terasoluna-tourreservation-web/src/test/java/org/terasoluna/tourreservation/app/reservetour/ReserveTourControllerTest.java

```diff
@@ -74,9 +74,8 @@ public void testReserveFormSuccess() {
                 "/tours/123/reserve").param("form", "");
 
         // Set mock behavior for helper method
-        when(reserveTourHelper.findTourDetail(
-                (ReservationUserDetails) anyObject(), eq("123"),
-                (ReserveTourForm) anyObject())).thenReturn(
+        when(reserveTourHelper.findTourDetail((ReservationUserDetails) any(),
+                eq("123"), any(ReserveTourForm.class))).thenReturn(
```

Comment:
any (Class) do not match `NULL` value and any () match `NULL` value .
And `ReservationUserDetails` parameter not setted in this test class until now .

I think ... above issue should resolve at another issue , if we want to test using `ReservationUserDetails` .
(Since this problem is involved in the testing of SpringSecurity, it is not appropriate to solve with this PR(Cairo Correspondence))

Reply:
#584 
I made new ISSUE .

Reply:
4e891c747a1bc25c8a5b907e8208ae2f83967cf2
I fixed to set UserDetails when testing ReserveTourController .